nilearn/image/tests/test_image.py

```diff
@@ -837,6 +849,55 @@ def test_math_img(
         assert result.shape == expected_result.shape
 
 
+def test_math_img_copied_header(
+    img_4d_ones_eye_default_header, img_4d_ones_eye_tr2, img_4d_mni_tr2
+):
+    img_eye_default = img_4d_ones_eye_default_header
+    img_eye_tr2 = img_4d_ones_eye_tr2
+    img_mni_tr2 = img_4d_mni_tr2
+
```

Comment:
Do we need this renaming ?

Reply:
Also maybe try to split this in 3 tests: one each case.

This way an eventual failure of the first assert would not prevent other cases to be run.

Reply:
also shorter test are easier to read, because you don't have to worry if something  written 30 lines higher still matters to keep in mind